swift_browser_ui/signature.py

```diff
@@ -29,12 +37,168 @@ async def handle_signature_request(
     ))
 
 
+async def handle_ext_token_create(
+        request: aiohttp.web.Request
+) -> aiohttp.web.Response:
+    """Handle call for an API token create."""
+    session = api_check(request)
+
+    project = request.app["Creds"][session]["active_project"]["id"]
+
+    LOGGER.debug(f"Creating a scoped API token for {project}")
+
+    ident = request.match_info["id"]
+    token = hashlib.sha256(os.urandom(256)).hexdigest()  # nosec
```

Comment:
I think this https://docs.python.org/3.7/library/secrets.html would be more useful here.